aiohttp/client.py

```diff
@@ -26,6 +26,8 @@
 from .helpers import (PY_35, CeilTimeout, TimeoutHandle, _BaseCoroMixin,
                       deprecated_noop, sentinel)
 from .http import WS_KEY, WebSocketReader, WebSocketWriter
+from .http_websocket import extensions_gen as ws_ext_gen
```

Comment:
Why do you rename imports?
Just use `ws_ext_gen` and `ws_ext_parse` in `http_websocket.py`.